test_rmw_implementation/CMakeLists.txt

```diff
@@ -34,6 +34,7 @@ if(BUILD_TESTING)
     )
     target_compile_definitions(test_init_shutdown${target_suffix}
       PUBLIC "RMW_IMPLEMENTATION=${rmw_implementation}")
+
```

Comment:
This added line can be removed

Reply:
Done in https://github.com/ros2/rmw_implementation/pull/119/commits/da381880ea605ff00ee45481e64b1759f2d57594.